packages/lib/src/components/internal/SecuredFields/lib/core/SecuredField.ts

```diff
@@ -78,9 +78,15 @@ class SecuredField extends AbstractSecuredField {
         this.config.iframeUIConfig.placeholders = processedPlaceholders;
 
         /**
-         * Create & reference iframe and add load listener
+         * Configure, create & reference iframe and add load listener
          */
-        const iframeEl: HTMLIFrameElement = createIframe(`${this.iframeSrc}`, processedAriaConfig[this.fieldType].iframeTitle);
+        const iframeConfig = {
+            src: this.iframeSrc,
+            title: processedAriaConfig[this.fieldType].iframeTitle,
+            policy: getSearchParameters().testing === 'testcafe' ? 'no-referrer-when-downgrade' : 'origin'
```

Comment:
What would you think of checking the `NODE_ENV` variable in this case, something like `process.env.NODE_ENV === 'test'?

Reply:
I don't know if this would work or not, since we are running the tests from the playground. So where do you set the NODE_ENV? Setting it in the package.json of playground doesn't seem to make any difference. We'd have to test.
And I don't know if this would work for the plugins team either

Reply:
Reverted this change since adding a <meta> tag fixes the issue for plugins - and always using `'origin'` works as long as you run components against Test (c.f. localhost)